### ISSUE TITLE
Realtime prediction

### ISSUE THREAD

**Issue by Zumbalamambo** [pedro-abreu/twostream-ava#1]
Thank you for the amazing work. Can you pease share the script for realtime prediction?

**Comment by pedro-abreu** [comment]
Can you please explain your question better?

**Comment by Zumbalamambo** [comment]
I mean, Can you please share with us the code to predict using my webcam which is by using cv2.VideoCapture(). Thank you.

**Comment by pedro-abreu** [comment]
I don't understand what you're asking. If you want code to predict a single instance, we did not implement that we merely have scripts to test on a test set for the AVA challenge.

**Comment by Zumbalamambo** [comment]
yes i want to predict on a single image.. Can you please provide some sample code for that?

**Comment by pedro-abreu** [comment]
We can provide prediction on a single segment of 3s video as this is a video detection scheme, but sure I'll do it, give me a couple of days :). (Leave a star on the repo haha)

**Comment by Zumbalamambo** [comment]
haha i have starred it the very first day when i stumbled across this awesome code!. its truly amazing. I have certain doubts as well which i would like to ask. Before that, Kudos to you for the great work. Thank you so much.  im learning many new things through  your code
